evm/rlp/headers.py

```diff
@@ -83,15 +86,15 @@ def __init__(self,
                  gas_limit: int,
                  timestamp: int=None,
                  coinbase: bytes=ZERO_ADDRESS,
```

Comment:
Should it be `coinbase: Address=ZERO_ADDRESS,` ?

Reply:
Thank you for spotting this. Fixed it!